internal/compiler/Cargo.toml

```diff
@@ -27,10 +27,13 @@ proc_macro_span = ["quote", "proc-macro2"]
 # Feature to print the diagnostics to the console
 display-diagnostics = ["codemap", "codemap-diagnostic"]
 
+# Enable Arbitrary support for fuzzing
+arbitrary = ["smol_str/arbitrary", "dep:arbitrary"]
```

Comment:
The `dep:` trick is only available from Rust 1.60 to be released in April.
You will have to find another way of you want this to be merged before we require it